src/_ert/threading.py

```diff
@@ -46,14 +55,17 @@ def run(self) -> None:
         try:
             super().run()
         except BaseException as exc:
-            logger.error(str(exc), exc_info=exc)
+            err_traceback = str(traceback.format_exc())
+            logger.error(err_traceback, exc_info=exc)
             if _can_raise and self._should_raise:
-                _raise_on_main_thread(exc)
+                _raise_on_main_thread(exc, err_traceback)
```

Comment:
Is there some special reason for getting the traceback here and passing it as a param instead of using exc.__traceback__ in ErtThreadErrror.__init__ and skipping the param? If it is possible to skip the param and keep the same behavior that'd would be ideal.

Reply:
>Is there some special reason for getting the traceback here and passing it as a param instead of using exc.traceback

Not sure what you mean by `exc.traceback` when I tried it I got `{AttributeError}AttributeError("'ValueError' object has no attribute 'traceback'")` there is something internal `exc.__traceback__` and I am not sure how to parse this object and extract the same information as I am getting from `traceback.format_exc()`. It might be possible but I am not sure it will be as straight forward as the current implementation. 

>  If it is possible to skip the param and keep the same behavior that'd would be ideal.

I think it is possible to skip the `err_traceback` param definition in `_raise_on_main_thread` I can try it.